auth/claims.go

```diff
@@ -30,7 +31,7 @@ func ParseClaims(token *oauth2.Token) (claims Claims, err error) {
 		return
 	}
 	jwt := strings.Split(token.AccessToken, ".")
-	payload, err := base64.StdEncoding.DecodeString(jwt[1])
+	payload, err := base64.RawStdEncoding.DecodeString(jwt[1])
```

Comment:
@manicminer the reason RawStdEncoding is required is because JWTs are not padded, and StdEncoding expects padding.

https://www.rfc-editor.org/rfc/rfc7515.html#section-2

```
   Base64url Encoding
      Base64 encoding using the URL- and filename-safe character set
      defined in Section 5 of RFC 4648 [RFC4648], with all trailing '='
      characters omitted (as permitted by Section 3.2) and without the
      inclusion of any line breaks, whitespace, or other additional
      characters.  Note that the base64url encoding of the empty octet
      sequence is the empty string.  (See Appendix C for notes on
      implementing base64url encoding without padding.)
```

Reply:
Thanks this makes sense. I didn't hit any encoding issues in testing presumably because my JWT payloads were short (minimal set of claims).

Reply:
@manicminer I think you have something in 1/4 to get padding or not. :+1: